enginetest/queries/insert_queries.go

```diff
@@ -2276,6 +2276,26 @@ var InsertScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "insert...returning... statements",
+		SetUpScript: []string{
+			"CREATE TABLE animals (id int, name varchar(20))",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "insert into animals (id) values (2) returning id",
```

Comment:
Might be cool to add a quick test that has an auto-generated primary key: either a pk with a default of `UUID()` or an `auto_increment` pk. The `returning` clause really shines when you insert something and want to know the generated ID, so would be cool to add a quick test for that.